File: Python/1491. Average Salary Excluding the Minimum and Maximum Salary.py
# 1491. Average Salary Excluding the Minimum and Maximum Salary
# Level: Easy
# https://leetcode.com/problems/average-salary-excluding-the-minimum-and-maximum-salary/

class Solution:
  def average(self, salary: List[int]) -> float:
    totalSalary = sum(salary)
    maxSalary = max(salary)
    minSalary = min(salary)

    return (totalSalary - maxSalary - minSalary) / (len(salary)-2)

File: PHP/1491. Average Salary Excluding the Minimum and Maximum Salary.php
<?php
// 1491. Average Salary Excluding the Minimum and Maximum Salary
// Level: Easy
// https://leetcode.com/problems/average-salary-excluding-the-minimum-and-maximum-salary/

class Solution
{
  function average($salary)
  {
    $totalSalary = array_sum($salary);
    $minSalary = min($salary);
    $maxSalary = max($salary);

    return ($totalSalary - $minSalary - $maxSalary) / (count($salary) - 2);
  }
}
